core/src/main/java/greencity/controller/ManagementNotificationController.java

```diff
@@ -0,0 +1,83 @@
+package greencity.controller;
+
+import greencity.annotations.ApiPageable;
+import greencity.constants.HttpStatuses;
+import greencity.dto.NotificationTemplateDto;
+import greencity.dto.PageableDto;
+import greencity.service.notification.ManagementNotificationService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import springfox.documentation.annotations.ApiIgnore;
+
+@RestController
+@RequestMapping("/admin/notification")
+@Validated
+public class ManagementNotificationController {
+    private final ManagementNotificationService notificationService;
+
+    /**
+     * Constructor with parameters.
+     */
+    @Autowired
```

Comment:
you don't need to declare constructor explicitly. Use @ RequiredArgsConstrutor lombok annotation instead